docs/standard/microservices-architecture/net-core-net-framework-containers/net-framework-container-scenarios.md

```diff
@@ -13,43 +13,37 @@ ms.topic: article
 
 While .NET Core offers significant benefits for new applications and application patterns, .NET Framework will continue to be a good choice for many existing scenarios.
 
-## Migrating existing applications directly to a Docker container
+## Migrating existing applications directly to a Windows Server container
 
 You might want to use Docker containers just to simplify deployment, even if you are not creating microservices. For example, perhaps you want to improve your DevOps workflow with Docker—containers can give you better isolated test environments and can also eliminate deployment issues caused by missing dependencies when you move to a production environment. In cases like these, even if you are deploying a monolithic application, it makes sense to use Docker and Windows Containers for your current .NET Framework applications.
 
-In most cases, you will not need to migrate your existing applications to .NET Core; you can use Docker containers that include the full .NET Framework. However, a recommended approach is to use .NET Core as you extend an existing application, such as writing a new service in ASP.NET Core.
+In most cases for this scenario, you will not need to migrate your existing applications to .NET Core; you can use Docker containers that include the full .NET Framework. However, a recommended approach is to use .NET Core as you extend an existing application, such as writing a new service in ASP.NET Core.
```

Comment:
still some references to full .NET Framework (more than one)

Reply:
👍